test/ShapeCrawler.Tests.Unit/PictureTests.cs

```diff
@@ -177,4 +178,41 @@ public void SendToBack_sends_the_shape_backward_in_the_z_order()
         // Assert
         shapes[0].Name.Should().Be("Picture 2");
     }
+
+    [TestCase("0")]
+    [TestCase("1")]
+    [TestCase("0.2")]
+    [TestCase("0.5")]
+    public void Transparency_setter_sets_expected_values(string expectedStr)
```

Comment:
@jcoliz, I've found that `string` parameter of `TestCase` is converted into `decimal`🙂 Please check this. For example, the following test works correclty:

```C#
[TestCase("0")]
[TestCase("1")]
[TestCase("0.2")]
[TestCase("0.5")]
public void Transparency_setter_sets_expected_values(decimal expected)
{
    // Arrange
    var pres = new Presentation(StreamOf("060_picture-transparency.pptx"));
    var picture = pres.Slides[0].Shapes.GetByName<IPicture>("50%");

    // Act
    picture.Transparency = expected;

    // Assert
    var actual = picture.Transparency;
    actual.Should().Be(expected);
}
```

Reply:
@ashahabov Oh wow. Today I learned! I wonder how long this has worked. I have written tests for years using this pattern.

f1729f54eb472db647a21529ac1108ece9da5335

Reply:
Me too🤷‍♂️ I've found this for xUnit on [StackOverflow](https://stackoverflow.com/a/37854575/2948684) and just try for NUnit.

Reply:
So now your challenge is to try and get this to work with `SlideShapeAttribute`. This did not work for me with simply changing the parameter type.

Reply:
For now I think it is not necessary to waste time on it. In the future we should learn at the implementation of `TestCaseAttribute` and "copy/past" such implementation in `SlideShapeAttribute`.